src/Steps/Steps.tsx

```diff
@@ -0,0 +1,17 @@
+import { Step1 } from "./Step1";
+import { FinalStep } from "./FinalStep";
+
+export const TMCRStepNames = ["Home", "TO Info", "TO Program Requirements", "TMSS Requirements", "IETM Functionaly Requirements", "Mandatory Requirements", "Configurable Requirements", "Delivery Requirements", "Spec/Std Interface Records (SIRS)", "Attachment 1", "Attachment 2", "Review/Print"];
+export const TMCRFinalStep = 11;
```

Comment:
@RobertPorterfield  - Should we base the TMCRFinalStep on the number of elements in the WizardSteps?  That way if the array changes (up or down) we don't have to worry about forgetting to update the max number.  Same with the TMCRStepNames -- would it be good to merge that into an array of Names/ReactObj?

Reply:
Agree. Current implementation is ugly. I'll add a small fix and a TODO to merge the steps and names into a single object. Next PR will define all the major steps and build out several of them., I'll include changes with that.